frontend/src/App.vue

```diff
@@ -34,17 +34,22 @@ html, body {
   --border-thick: 2px solid rgba(160, 178, 178, 0.5);
   --box-shadow-active: 0px 0px 0px 1px #ddd;
   --theme: #00a99d;
-  --theme-light: rgba(#00a99d, 0.6);
+  /*--theme-light: rgba(#00a99d, 0.6);*/
+  --theme-light: rgba(102, 203, 196);
   --font-large: 40px;
   --font-large-mobile: 24px;
   --font-small: 16px;
   --font-small-mobile: 12px;
 }
 
-main {
+#app {
+  background-color: var(--color-light-gray);
+}
+
+.main-content {
```

Comment:
not sure what I think of this XD.

Reply:
Maybe it makes sense I guess. Only, main is already meant to be main content.

Reply:
alternative is to put a main tag in each view

Reply:
Purpose of the main tag is to be the view container though, where header and footer are agnostic of view.